sass/elements/blocks/_blocks.scss

```diff
@@ -33,7 +38,8 @@ hr {
 	border-image-slice: 4 4 4;
 	border-image-width: 0 0 4px;
 	border-image-repeat: round;
-	margin: spacing(3) 0;
+	margin: spacing(3) auto;
+	max-width: 600px;
```

Comment:
Any reason for not using `$size__width--post` to make this 620px?

With 600px, the wide and normal dots don't align on the left and right. Given it's only noticeable when those two separators are used close to each other I'm not worried about it if it fixes something else. 

![image](https://user-images.githubusercontent.com/4986487/56937384-e84b6b80-6ac1-11e9-85d6-c7388fc94b98.png)

Reply:
They should all be 600, I guess – the issue is due to the way the dots are resized, when they're 620, the resize makes it uneven so we end up with the split dots (if you look close at my screenshot you'll see it, some dots look 1px narrower, though it's only obvious on non-retina displays). I think the value needs to be divisible by 8 (or maybe 12), so 600 works, but 620 doesn't.

Reply:
Got it. Makes sense to me.